Packs/IPQualityScore/pack_metadata.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "IPQualityScore (IPQS) Threat Risk Scoring",
+  "description": "Detect threats with real-time risk scoring by IPQS. Playbook analyzes IP addresses, email addresses, and domains or URLs for high risk behavior.",
+  "support": "partner",
+  "currentVersion": "1.0.0",
+  "author": "IPQualityScore",
+  "url": "https://www.ipqualityscore.com",
+  "email": "support@ipqualityscore.com",
+  "created": "2021-03-31T00:00:00Z",
+  "categories": [
+    "Data Enrichment & Threat Intelligence",
+	"Network Security"
+  ],
+  "tags": ["Attack", "Dark Web", "Email", "Malware", "Network", "Scam", "Security", "Security Analytics", "Spam", "VPN", "Threat Intelligence"],
+  "useCases": ["Brute Force", "Hunting", "Malware", "Phishing", "SANS", "Threat Intelligence Management", "Network Security", "Compliance", "Reputation"],
+  "keywords": ["risk score", "threat score", "botnet", "bad actor", "ATO", "takeover", "account", "cyber", "proxy", "vpn", "tor", "ip intelligence", "ip address", "ip", "domain scoring", "parked", "parking", "domain age", "abuse", "threat", "risk", "fraud", "email validation", "url scanner", "malicious url", "malicious domain", "phishing", "malware", "ip reputation", "email reputation", "domain reputation", "url reputation", "detect", "prevent", "domain", "url"]
+}
```

missing "certification" field:
```suggestion
  "keywords": ["risk score", "threat score", "botnet", "bad actor", "ATO", "takeover", "account", "cyber", "proxy", "vpn", "tor", "ip intelligence", "ip address", "ip", "domain scoring", "parked", "parking", "domain age", "abuse", "threat", "risk", "fraud", "email validation", "url scanner", "malicious url", "malicious domain", "phishing", "malware", "ip reputation", "email reputation", "domain reputation", "url reputation", "detect", "prevent", "domain", "url"],
  "certification": "certified"
}
```